libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -103,8 +131,34 @@ void AC_Fence::enable(bool value)
     }
     _enabled = value;
     if (!value) {
-        clear_breach(AC_FENCE_TYPE_ALT_MAX | AC_FENCE_TYPE_CIRCLE | AC_FENCE_TYPE_POLYGON);
+        clear_breach(AC_FENCE_TYPE_ALT_MIN | AC_FENCE_TYPE_ALT_MAX | AC_FENCE_TYPE_CIRCLE | AC_FENCE_TYPE_POLYGON);
+    } else {
+        // turn the floor back on if it had been off
```

Comment:
Not making any sense of this comment.  Seems unconditional here....

Reply:
As you said: 
> These comments always get out of date.

Carried across from the `geofence_set_enabled` function. Can remove.